client/src/components/fortune.vue

```diff
@@ -65,3 +65,22 @@ export default {
   }
 };
 </script>
+
+<style scoped>
+h1 img {
+  display:inline;
+}
+th:nth-child(1){
+   width: 50%;
+}
+th:nth-child(2){
+   width: 20%;
+}
+th:nth-child(3){
+   width: 10%;
+}
+th:nth-child(4){
```

Comment:
Not a blocker or a critique but incase you didn't know you can group things together that have the same attributes. for example:

```
th:nth-child(2), th:nth-child(4) {
    width: 20%;
}
```

Reply:
I totally forgot about this!